csrankings-m.csv

```diff
@@ -2086,6 +2088,7 @@ Minsoo Rhu,KAIST,https://sites.google.com/view/kaist-via,u_3ShigAAAAJ
 Minsoo Ryu,Hanyang University,http://rtcc.hanyang.ac.kr/rtccw/?page_id=27,NOSCHOLARPAGE
 Minsu Cho,POSTECH,http://cvlab.postech.ac.kr/~mcho,5TyoF5QAAAAJ
 Minsuk Kahng,Oregon State University,https://minsuk.com,vieI1GYAAAAJ
+Minwoo Lee,UNC - Charlotte,https://webpages.uncc.edu/mlee173/,awrLUJ4AAAAJ

```

Comment:
That is a disambiguation entry. Please point to the correct DBLP name. https://dblp.org/pid/37/8786.html

Reply:
Min Shin had 2 of 3 publications associated with the Min C. Shin at UNC - Charlotte. I requested dblp to correct this.

Reply:
I deleted Minwoo Lee. I will ask dblp to create entry for him.